jmclient/jmclient/wallet_utils.py

```diff
@@ -472,12 +507,15 @@ def get_addr_status(addr_path, utxos, is_new, is_internal):
         elif len(status) == 1:
             out_status = status[0]
 
+        out_status += get_utxo_status_string(utxos, utxos_enabled, addr_path)
```

Comment:
So this means that if an address is reused, and one of the utxos is frozen, it will say 'reused [FROZEN]' right? If I recall correctly, our current logic is to auto-freeze (by default configuration) new utxos that are added to an already used address, but it's possible for a user to simply deposit twice into the same address, where one utxo will be available for spending (the first) and the other(s) won't.

I'm not saying this is wrong necessarily, but it needs to be considered.

Reply:
> So this means that if an address is reused, and one of the utxos is frozen, it will say 'reused [FROZEN]' right?

Yes.

> it's possible for a user to simply deposit twice into the same address, where one utxo will be available for spending (the first) and the other(s) won't.

Correct. And I think the current behavior makes sense. If there are two UTXOs in the same address, and one is frozen while the other isn't, we have have two options for that address: either show "[FROZEN]" or do not show it. Even though neither options are perfectly accurate, I think showing [FROZEN] is the better choice here.

Reply:
Right. I do get the logic and I think I agree, I just think it's suboptimal, in the specific sense that we show x(y) at the bottom indicating y-x is available, but actually more is available.